uktrade/data-hub-frontend
# Cls2 483 my tasks reminders

## Description of change

My tasks reminders section added to reminders pages with due date approaching reminders. 
Also added tasks section to the reminders panel in the dashboard.

The notification bell for reminders is now always visible. Before it was only visible if users had either feature flag (investments/exports reminders)

## Test instructions

With the sandbox environment running:
Navigate to `/reminders` 
You should see `My Tasks` in the left hand panel with `Due Date Approaching`
Select this to see example sandbox task reminders

With the dev environment running:
Navigate to `/reminders` 
You should see `My Tasks` in the left hand panel with `Due Date Approaching`
Add `upcoming investment project task reminder` in the django backend and they should appear here

## Screenshots

### Before
![Screenshot 2023-10-19 at 15 23 15](https://github.com/uktrade/data-hub-frontend/assets/54268863/564ccc23-a6ec-4a98-9375-dab924d01d42)

### After
![Screenshot 2023-10-24 at 15 06 32](https://github.com/uktrade/data-hub-frontend/assets/54268863/f8b3476b-6a14-4b73-b84c-473dfecaf77b)
(This is an older image)
![Screenshot 2023-10-19 at 15 22 38](https://github.com/uktrade/data-hub-frontend/assets/54268863/fd64d99d-baa1-4b91-b860-c183b5bbc0d2)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
